src/jsMain/kotlin/app/softwork/bootstrapcompose/Navbar.kt

```diff
@@ -0,0 +1,174 @@
+package app.softwork.bootstrapcompose
+
+import androidx.compose.runtime.*
+import kotlinx.uuid.*
+import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.dom.*
+import org.w3c.dom.*
+
+/**
+ * Bootstrap Navbar component. This version provides more flexibility than the alternative but is more complex
+ * to use. For a simpler implementation use the version that takes a brand and navItems as parameters.
+ *
+ * @param breakpoint Specifies the navbar-expand breakpoint
+ * @param fluid Specifies if the inner container is fluid (container-fluid) or not.
+ * @param containerBreakpoint Breakpoint for the inner container.
+ * @param colorScheme Valid values are Color.Light or Color.Dark to set the navbar-dark/light class.
+ * @param backgroundColor Background color to use, with the bg-* classes.
+ * @param content Navbar content, placed within the inner container.
+ */
+@Composable
+public fun Navbar(
+    breakpoint: Breakpoint = Breakpoint.Medium,
+    fluid: Boolean = false,
+    containerBreakpoint: Breakpoint? = null,
+    colorScheme: Color = Color.Light,
+    backgroundColor: Color = Color.Primary,
+    attrs: AttrsBuilder<HTMLElement>.() -> Unit = { },
+    content: ContentBuilder<HTMLElement>? = null
```

Comment:
To be consistence with this wrapper library, please use non-null `ContentBuilder` and use `{ }` as a default value.